core/src/main/resources/reference.conf

```diff
@@ -70,6 +70,9 @@
         port = 8125
         period = 10 seconds
         prefix = snowplow.collector
+        tags = {
```

Comment:
In most other apps we leave the default tags empty, e.g. [here](https://github.com/snowplow-incubator/snowplow-lake-loader/blob/0.1.4/modules/core/src/main/resources/reference.conf#L33).

Reply:
I used nonempty default, because that's what we do for [old](https://github.com/snowplow/stream-collector/blob/fd41ec04665a8dfd68dc7d7e610d49556a35e96d/core/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/model.scala#L181) collector?

Reply:
True... but we never yet used those metrics internally.  Whereas now there is a plan to start using them.  So now is our chance to get the tags right.

If we forward these metrics to cloudwatch, then each tag becomes a cloudwatch Dimension.  And for cloudwatch you must specify every Dimension when querying a meric.  So by adding a default tag here, it affects how the metrics get queried later.  By leaving tags empty, we can be consistent across all apps when it comes to querying the metrics.